irr/src/CGLTFMeshFileLoader.cpp

```diff
@@ -305,25 +305,27 @@ SelfType::createNormalizedValuesAccessor(
 	}
 }
 
-template <std::size_t N>
+template <std::size_t N, bool validate>
 std::array<f32, N> SelfType::getNormalizedValues(
 		const NormalizedValuesAccessor<N> &accessor,
 		const std::size_t i)
 {
 	std::array<f32, N> values;
 	if (std::holds_alternative<Accessor<std::array<u8, N>>>(accessor)) {
 		const auto u8s = std::get<Accessor<std::array<u8, N>>>(accessor).get(i);
-		for (u8 i = 0; i < N; ++i)
-			values[i] = static_cast<f32>(u8s[i]) / std::numeric_limits<u8>::max();
+		for (u8 j = 0; j < N; ++j)
```

Comment:
is there any reason to use such a narrow type here? shouldn't this be `size_t`?

Reply:
It's pretty much the smallest type that fits here, we don't need a `size_t`. `N` is something like `2` or `4`, definitely way below `255`. Though I just noticed that it is a bit inconsistent, the template has `size_t N`. I could either make both `u8` or make both `size_t`. In the end it doesn't really matter, it's just a very minor style question. I think I'll go with `size_t`.

Reply:
https://github.com/minetest/minetest/pull/15310/commits/6a28d9b6a7cc6bfdaf2b8028de53150f3d91d972